low-level/framework/utils/utility.py

```diff
@@ -58,3 +60,34 @@ def is_env_vm(self):
             logger.warning("Error while reading whether env is vm or not, assuming VM env : {e}")
         return is_vm
 
+    @staticmethod
+    def _initialize_dbus():
```

Comment:
keep dbus api in cortx-utils and use from there. Dont duplicate code here

Reply:
Done